pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[tool.poetry]
+name = "eq-questionnaire-schemas"
+version = "0.1.0"
+description = "Questionnaire Schemas Repository"
+authors = ["ONSDigital"]
+
+
+[tool.poetry.dependencies]
+python = "^3.10"
+coloredlogs = "*"
+black = "*"
+isort = "*"
```

Comment:
These should be dev dependencies. Worth adding pylint.
Can you also make sure these are added to the Makefile and Actions.

Reply:
I've added pylint (fixed the warnings raised) and a lint step for the actions 👍